mining.go

```diff
@@ -1158,23 +1159,11 @@ mempoolLoop:
 			continue
 		}
 
-		// Fetch all of the transactions referenced by the inputs to
-		// this transaction.  NOTE: This intentionally does not fetch
-		// inputs from the mempool since a transaction which depends on
-		// other transactions in the mempool must come after those
-		// dependencies in the final generated block.
-		txStore, err := blockManager.FetchTransactionStore(tx, treeValid)
-		if err != nil {
-			minrLog.Warnf("Unable to fetch transaction store for "+
-				"tx %s: %v", tx.Sha(), err)
-			continue
-		}
-
 		// Need this for a check below for stake base input, and to check
 		// the ticket number.
-		isSSGen, _ := stake.IsSSGen(tx)
-
-		if isSSGen, _ := stake.IsSSGen(tx); isSSGen {
+		txType := txDesc.Type
+		isSSGen := txType == stake.TxTypeSSGen
```

Comment:
I know this is legal go, but I kind of think lines like that are very hard to read when going through it quickly.  Is there any clearer way to do it?